src/euphorie/client/templates/shell.pt

```diff
@@ -271,11 +271,15 @@
                      data-pat-switch="selector: body; remove: focus-*; add: focus-document &amp;&amp; selector: body; remove: sidebar-on; add: sidebar-off"
                      tal:condition="not:disabled"
                      i18n:translate="navigation_status"
-                  >Status</a>
+                  >Status
+                    <strong class="pie-graph" data-percentage="${webhelpers/traversed_session/session/completion_percentage|string:0}" title="${webhelpers/traversed_session/session/completion_percentage|string:0}% Complete">${webhelpers/traversed_session/session/completion_percentage|string:0}%</strong>
```

Comment:
i18n needs to be handled, so that a German user will see "69% Komplett" instead of "69% Complete" etc

Reply:
Hehehe, I just realised, this whole tag is not valid:
Note that the outer `<a>` tag already has an `i18n:translate` statement. In tools of a different language, the percentage indicator simply gets overwritten by the translation :)
I think the inner `<strong>` either needs to be escaped via i18n:name, or you place simple tag like `<tal:i18n i18n:translate="navigation_status">` only around the label “Status”.